articles/storage/common/storage-import-export-service-faq.md

```diff
@@ -82,9 +82,18 @@ You can ship drives to the Azure datacenter using any known carrier like FedEx,
 - A FedEx account number in the US and EU, or
 - A DHL account number in the Asia and Australia regions.
 
+Note: India Datacenters require a Declaration letter on customer’s letterhead for returning the drives (Delivery Challan). You must also book the collection of the disks with your selected carrier and share the pickup details with the Datacenter in order to arrange the required entry pass. 
+
 ### Are there any restrictions with shipping my drive internationally?
 Please note that the physical media that you are shipping may need to cross international borders. You are responsible for ensuring that your physical media and data are imported and/or exported in accordance with the applicable laws. Before shipping the physical media, check with your advisors to verify that your media and data can legally be shipped to the identified data center. This will help to ensure that it reaches Microsoft in a timely manner.
 
+### Are there any special requirements for delivering my disks to a DC?
+
+It depends on some specific Azure Datacenter restrictions.
```

Comment:
Change the text to:
The requirements depend on some specific Azure datacenter restrictions. 

- There are few Class C sites (for example, Frankfurt), that require a Microsoft datacenter Inbound ID number to be written on the parcel for security reasons. Before you ship your drives/disks to the datacenter, reach out to Azure Data Box Operations alias *adbops@microsoft.com* to get this number. Without this number, the package will be rejected.  
- The datacenters in India require the personal details of the driver such as the Government ID Card or Proof No. (for example, PAN, AADHAR, DL), name, contact, and the car plate number to get a gate entry pass. To avoid delivery delay, make sure to inform your carrier regarding these requirements.